### PR TITLE
Unpack ROMs to bins before building

### PR DESCRIPTION
**Pull request by jjshoots**
This PR changes the implementation such that ROMs are used in the build rather than the tar.gz as requested by @JesseFarebro in #516

### CODE REVIEW

**Comment by jjshoots** [comment]
@pseudo-rnd-thoughts ready for review + merge

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -105,7 +105,7 @@ jobs:
         doNotCache: true
 
     - name: Download tar.gz.b64
```
Comment (pseudo-rnd-thoughts) [inline .github/workflows/ci.yml:107]: Jesse requested this name be improved

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ jobs:
       run: |
         unzip wheels.zip -d dist
     - name: Download tar.gz.b64
```
Comment (pseudo-rnd-thoughts) [inline .github/workflows/release.yml:21]: Same here

File: download_unpack_roms.sh
Comment (pseudo-rnd-thoughts) [inline download_unpack_roms.sh:1]: Jesse suggested that we move this to `.github` or `scripts`, basically out of root. 
We can add a note for developers about this for testing locally

File: src/python/roms/tetris.bin
Comment (pseudo-rnd-thoughts) [inline src/python/roms/tetris.bin:1]: Why has this changed?

**Review by pseudo-rnd-thoughts — CHANGES_REQUESTED** [review]
Otherwise looks good

File: src/python/roms/tetris.bin
Comment (jjshoots) [reply src/python/roms/tetris.bin:1]: Python's unpack and bash's unpack produces fundamentally different sized files, but the checksums still pass: https://stackoverflow.com/questions/69585800/what-is-the-fundamental-difference-between-tar-unix-and-tarfile-python

**Comment by jjshoots** [comment]
@pseudo-rnd-thoughts done.

**Comment by jjshoots** [comment]
Ah shit something broke.

**Review by pseudo-rnd-thoughts — APPROVED** [review]

**Pull request merged**